src/pages/invoice/create.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../../styles/common.css" />
+    <link rel="stylesheet" href="../../styles/invoice.css" />
+    <script defer src="/script/main.js"></script>
+    <title>Billbook | Create New Invoice</title>
+</head>
+
+<body class="dashboard" style="overflow-y: auto;">
+  <nav class="nav-bar" id="nav-bar">
+    <ul>
+      <a href="/pages/customers">
+        <li class="nav-item"><i class="material-icons">person</i> Customers</li>
+      </a>
+      <a href="/pages/items">
+        <li class="nav-item"><i class="material-icons">star</i>Items</li>
+      </a>
+      <a href="#">
+        <li class="nav-item nav-item--active">
+          <i class="material-icons">description</i>
+          Invoices
+        </li>
+      </a>
+    </ul>
+  </nav>
+
+  <nav class="bottom-nav">
+    <ul>
+      <li class="bottom-nav-item">
+        <a href="/pages/customers"><i class="material-icons">person</i></a>
+      </li>
+      <li class="bottom-nav-item">
+        <a href="/pages/items"><i class="material-icons">star</i></a>
+      </li>
+      <li class="bottom-nav-item bottom-nav-item--active">
+        <a href="#"><i class="material-icons">description</i></a>
+      </li>
+      <li class="bottom-nav-item">
+        <button onclick="showNavBar(event)"><i class="material-icons">menu</i></button>
+      </li>
+    </ul>
+  </nav>
+
+  <main class="content">
+    <header class="content__header">
+      <h1>New Invoice</h1>
+      <button class="btn btn-primary">
```

Comment:
Is there a `btn-primary` class?

Reply:
It's not but it seems it's there in other files too. Good eye.

Reply:
Done. Check this out [4a9bf89](https://github.com/ashrivas8/rz-billbook/pull/17/commits/4a9bf89a86ecdd51376f026e959360123c51497f) .